Newsletter: Quarterly planning mistakes to avoid

ivanagas opened the pull request:
## Changes

First draft of upcoming newsletter on quarterly planning. Would love feedback on:

1. Is the frame of the piece right? I feel like mistakes to avoid works well here, but also considers
  - What engineers get wrong about quarterly planning
  - Non-obvious quarterly planning advice for engineers 
3. Any points I'm missing?
4. Any points you disagree with or shouldn't be included?
5. Anything I should add to 5 and 6?
6. Are they in the right order?
7. Any examples that could be helpful here? 

We can save more detailed copy feedback for later. 

## Checklist

- [ ] Words are spelled using American English
- [ ] Titles are in [sentence case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case)
- [ ] Feature names are in **[sentence case too]([https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case))**
- [ ] Use relative URLs for internal links

## Article checklist

- [ ] I've added (at least) 3-5 internal links to this new article
- [ ] I've added keywords for this page to the rank tracker in Ahrefs
- [ ] I've checked the preview build of the article
- [ ] The date on the article is today's date
- [ ] I've added this to the relevant "Tutorials and guides" docs page (if applicable)

edwinyjlim commented:
> Is the frame of the piece right? I feel like mistakes to avoid works well here, but also considers
> - What engineers get wrong about quarterly planning
> - Non-obvious quarterly planning advice for engineers

My main feedback is on framing. IMO it’s a little difficult for the reader to place themselves in this piece right now. Company planning involves several personas – founders, team leads, and ICs – and the newsletter kinda floats between them. It's not always clear which role or at what org level each section is addressing. For example, `Being top down` is clearly aimed at execs and founders, but `Not being clear about what success looks like` feels like it could apply to anyone (execs, ICs, team leads, small teams, big teams, etc.)

I think this gets resolved if you clarify an explicit perspective or level of zoom within each h2 section. That way readers can orient themselves on what advice they can directly or indirectly apply

**Some ideas and suggestions:**

1. H1 title: `Quarterly planning mistakes to avoid **as a startup**` or `How we do quarterly planning at PostHog`

Your current framing basically, which is advice at the company-wide level. After reading your draft, I kind of like going with something like `How we do quarterly planning at PostHog`. It feels less prescriptive and emphasizes the behind-the-scenes angle, which is how it comes across to me right now

I think `Mistakes to avoid` angle totally works too. But I think the current draft needs more specific, concrete examples of each mistake or what signs to watch for when planning has gone wrong

2. H1 title: `What engineers get wrong about quarterly planning` or `Non-obvious quarterly planning advice for engineers`

If we go down this path, we could potentially create two newsletters: one for product engineers and one for founders (`What execs get wrong about quarterly planning`). I think there's enough material to make them both interesting!

3. There's an old saying that I like if you wanna use it, `Plans are useless, but planning is indispensable`.

ivanagas commented:
Ok, big rewrite, thanks for all the feedback. Tried to address all your points. 
- Moved around some points
- Made it much more engineering focused and actionable for engineers specifically.
- Although some companies don't do planning on a quarterly basis, enough of them do that I feel like this frameworks. There are a bunch of concepts (OKRs, QBRs) that are a product of quarterly planning. Also, we do it. For the smaller number of people who don't do quarterly planning, I feel like they will still get what we mean. Just "planning" on its own feels a bit vague and weak IMO. 
- Tried to add more examples to W framework planning process
- Changed objectives -> goals in many places
- I did think about including growth reviews but intentionally left it out as it doesn't connect super strongly to quarterly goals.

edwinyjlim commented:
Love the updated draft  

Here's my messy note on how I'd potentially sequence these sections in terms of flow:

- H2: `Thinking you need the perfect plan` (i.e., level-setting expectations for planning)
- H2: ~`Planning the whole quarter in advance`~
	- This section feels a little thin to me. I think the core point on control is already captured by `Thinking you need the perfect plan` and "not being afraid to change goals"
	- I really like the numbered list of issues
	- I think a condensed version could be folded into the `Thinking you need the perfect plan` section
- H2:  `Skipping the prep work` or `Doing it live` (i.e., preparation before planning is crucial)
	- My proposal to replace `Planning the whole quarter in advance`
	- I think we have a lot of opinions on doing the prep work before planning. [Raquel](https://posthog.slack.com/archives/C0351B1DMUY/p1750094565836269), [Andy](https://docs.google.com/document/d/1MACgX4vre6mz0KloNnXNK__0scZHpWRWQACm9nhGT-M/edit?tab=t.0#heading=h.7438q7gwie9u), and our [handbook](https://posthog.com/handbook/company/goal-setting) have some great talking points on planning prep
	- Reintroduce HOGS template and how it works for us to into the planning mindset
	- "These meetings aren't for individual thinking - they're primarily for discussion about the individual thinking that's already been done"
- H2: `Obsessing over metric-based OKRs `(i.e., don't force metrics)
- H2: `Being unclear about what success looks like` (i.e., you still need a clear target)
- H2: `Ignoring your plan after writing it` (i.e., regular check-ins need to happen)
- H2: `Letting others own your plan` (i.e., cross-company, cross-level collaboration)
	- I think this section should come last, because it's the most involved in terms of company process

Lior539 reviewed (commented):
overall looks good!

ivanagas commented:
@andyvan-ph now is a good time to take a look if you want...

andyvan-ph reviewed (commented):
This is looking good. Left some inline suggestions. Some higher-level bits:

- Doing it all live and the OKR points feel like the weakest atm. They're good points, they just need a bit more meat to make them feel real an actionable.

- This is my goto suggestion ;), can we do some boxout takeaways / do this style boxouts at the end of each section?

- Is there a meta point to end on that's like "Don't copy your process directly from somewhere else". The unmentioned thing from the OKR section is copying a process because another company does it this way (e.g. Google) sucks. We can say the same about us. These are the lessons we've learned and you're free to copy them, but remember to create away that works for you, than just blindly copying someone else's framework.

Lior539 commented on contents/newsletter/quarterly-planning-mistakes.md:
```diff
@@ -0,0 +1,154 @@
+
+The second lesson is **not being afraid to change goals** midway through the quarter. Sometimes we get our goals wrong and that's ok. Circumstances change, delays happen, engineers need the freedom to adjust.
+
+![Changing goals](https://res.cloudinary.com/dmukukwp6/image/upload/2025_06_13_13_13_42_6834ffc060.jpg)
```
I think we should aim to hogify all memes in all newsletters going forward
